pages/checkout/access.tsx

```diff
@@ -119,7 +123,7 @@ const CheckoutAccessView = ({ assetId }: CheckoutViewProps) => {
 export const getServerSideProps: GetServerSideProps = async (context) => {
   const { id } = context.query
   // @TODO when we have server side token handle the case of no id or no data
-  return { props: { assetId: id || null } }
+  return { props: { assetId: id || undefined } }
```

Comment:
isn't `id || undefined` a little redundant?

Reply:
Hmm, I actually changed it back to `null` now since it was too easy to run into issues with SSR and serializing JSON. However I guess we will improve and change things here when we test SSR data fetching
![Skjermbilde 2022-08-24 kl  14 38 48](https://user-images.githubusercontent.com/1568252/186421204-70b692c8-736a-455b-a9ad-f0934b6cdfa2.png)

Reply:
oooh true. when serializing JSON, `undefined` values are omitted